Makefile

```diff
@@ -45,7 +45,6 @@ install-sys-requirements: install-system install-node
 	npm config set loglevel warn
 
 install-dev:
-	gem install sass
```

Comment:
This was the line that kept erroring out on fresh devstacks - it's included as part of `make install`, and `gem` is no longer installed on the box by default.